docs/docs/integrations/importers/launchdarkly.md

```diff
@@ -10,23 +10,8 @@ You can import your Flags and Segments from LaunchDarkly into Flagsmith.
 ## Integration Setup
 
 1. Create a LaunchDarkly Access Token. In LaunchDarkly: Account settings > Authorization > Access tokens.
-2. Using your Access Token and a LaunchDarkly project key (usually `"default"`) , create an Import Request for a
-   Flagsmith project of your choice:
-
-```bash
-curl -X 'POST' \
-  'https://api.flagsmith.com/api/v1/projects/<project_id>/imports/launch-darkly/' \
-  -H 'content-type: application/json' \
-  -H 'authorization: Token <api token>' \
-  -d '{"token":"<launchdarkly token>","project_key":"default"}'
-```
-
-4. The import will begin immediately. Check the import request status:
-
-```bash
-curl https://api.flagsmith.com/api/v1/projects/<project id>/imports/launch-darkly/<import_request_id>/ \
-    -H 'authorization: Token <api token>'
-```
+2. Create a new Project within Flagsmith, then go to Project Settings > Import. Add your Token generated in step 1.
+3. The import will begin immediately. Please don't navigate away from the page until the import is complete.
```

Comment:
Technically incorrect, the loading page only polls the import status. The import will proceed regardless of the actions in UI at this point.